src/main/java/com/nike/cerberus/command/core/AddCertificateToAlbCommand.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2017 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.command.core;
+
+import com.beust.jcommander.Parameters;
+import com.nike.cerberus.command.Command;
+import com.nike.cerberus.operation.Operation;
+import com.nike.cerberus.operation.core.AddCertificateToAlbOperation;
+
+import static com.nike.cerberus.command.core.AddCertificateToAlbCommand.COMMAND_NAME;
+
+@Parameters(
+        commandNames = {
+                COMMAND_NAME
+        },
+        commandDescription = "Adds a new cert to the alb"
+)
+public class AddCertificateToAlbCommand implements Command {
```

Comment:
Can the ALB contain more than one certificate?

If not, then I'm not sure this command is needed. You should be able to update the ALB stack with the new IAM certificate ARN for CMS.

Reply:
The ALB does support more than one cert or rotating would be impossible without downtime, but this is just a stub, might change it around.

Reply:
Cool!